package.json

```
@@ -1,18 +1,20 @@
 {
   "name": "dagre-d3-es",
-  "version": "7.0.11",
+  "version": "7.1.11",
   "description": "",
   "author": "Thibaut Lassalle",
   "license": "MIT",
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "type": "module",
   "scripts": {
-    "generate_types": "tsc --build --clean && tsc --build && prettier --write .",
-    "prepack": "npm run generate_types",
+    "dev": "tsc --watch",
+    "prepublishOnly": "rm -rf dist dist_tmp; mkdir dist; tsc --build tsconfig.json --declaration && mv dist_tmp/src/* dist && rm -rf dist_tmp",
```

This should probably be a [`"prepare"` script](https://docs.npmjs.com/cli/v10/using-npm/scripts#life-cycle-scripts) instead so that it runs automatically when somebody does `npm install https://github.com/tbo47/dagre-es`.

---

Also, we could simplify this to just:

```suggestion
    "prepare": "tsc --build tsconfig.json --declaration --clean",
```

if you modify your `tsconfig.json` like:

```json
{
  "include": [
    "src/**/*"
  ],
  "exclude": [
    "src/**/*.test.ts"
  ],
  "compilerOptions": {
    "target": "ES6",
    "rootDir": "src/",
    "outDir": "dist/",
    "lib": ["dom", "dom.iterable", "es2020"],
    "types": ["@types/jest"],
    "module": "node16",
    "skipLibCheck": true
  }
}
```

---

Thanks for reviewing!
I tried your suggestion (including the change in tsconfig.json) and it doesn't generate anything in the dist/ folder. So I will update the `prepublishOnly` to `prepare` for now. Don't hesitate to suggest something else.